tsnet/tsnet.go

```diff
@@ -566,8 +574,9 @@ func (s *Server) start() (reterr error) {
 	lal := memnet.Listen("local-tailscaled.sock:80")
```

Comment:
does anything here need to be cleaned up? I would've expected closing this to shut down the server

Reply:
Reading the docs on `(net.Listener).Close` suggests that closing existing conns is not a part of it. It only stops new connections from being established. So if there is a persistent albeit idle connection open, we need to call `server.Shutdown` to close them.